packages/record_thumbnail_attacher/src/index.test.ts

```diff
@@ -48,11 +48,17 @@ describe("handler", () => {
           messageId: "1",
           receiptHandle: "1",
           body: JSON.stringify({
-            s3: {
-              object: {
-                key: "_Liam/access_copies/e38e/8582/b417/430c/953d/5c7e/8040/1ae2/2_upload-cb45fa84-f0ea-4a9e-b1da-309e485a4f4a/object/710a1def-caf8-48f2-8eee-0848b4cfda10.jpg",
-              },
-            },
+            Message: JSON.stringify({
```

Comment:
Are these test change related to the infra as code changes or is it fixing a separate issue?

It may be worth putting them in their own commit just to contextualize the reason for the change.